data/avgi/map planets.txt

```diff
@@ -0,0 +1,539 @@
+	description "	Despite the constant rains and frequent storms, the planet's bountiful oceanic ecosystem is able to sustain a small community of particularly stubborn Avgi, who managed to eke out a life on a rocky island chain. They refused to be evacuated in the wake of the Incursion, and their outpost remains, weathering the wind and sea."
+	spaceport "The 'spaceport' is nothing more than a stony island mostly cleared of boulders and debris. A few small cranes and fuel tanks are strewn haphazardly along the edge, used to service the rare visiting starship."
+	government Avgi
+	security 0
+
+
+planet Rolion
+	attributes "gas giant" "requires: gaslining" uninhabited
+	landscape land/nasa18
+	description "At some point in the recent past, Rolion had a close encounter with another massive planet - potentially another gas giant, which was subsequently ejected from the Mellow system. Whatever the details, the encounter tipped Rolion on its side and completely upturned its atmosphere, creating wildly surging swells and storms larger than most terrestrial planets. The gravitational interaction between the two planets also heated the gas giant quite significantly, and in the infrared spectrum Rolion glows brightly."
+	government Uninhabited
+	security 0
+
+planet Orodens
+	attributes moon mining
+	landscape land/mountain26
+	description "This moon is speculated to have once been much larger, until some event stripped it of its crust and mantle. Now, only the metal rich core remains, serving as an alluring target abundant in heavy metals. More recently, Orodens is thought to have supported a hydrological cycle for a brief period, creating rich veins of precipitated minerals. However, the high surface gravity made it costly to transport material to orbit, where it could be shipped to other system. Before the recent invention of the gravity repulsor, rows of mass drivers fired streams of raw material into orbit, where it would be collected by waiting ships."
+	description "This was once the largest single source of fissile elements in Avgi space, but since the Incursion the mass drivers have gone quiet. Automated mining drones continue to watch over the last batch of excavated material."
+	government Uninhabited
+	security 0
+
+
+planet Pil
+	attributes moon uninhabited
+	landscape land/mars1
+	description "This airless moon is notable only for its rich deposits of light elements such as lithium and phosphorus. Their concentration was high enough to economically justify bringing them up out of the moon's gravity well, instead of simply mining the carbonaceous asteroids present in the system."
```

Comment:
```suggestion
	description "Pil is an airless moon notable only for its rich deposits of light elements, such as lithium and phosphorus. Their concentration was high enough to economically justify the Avgi lifting them out of the gravity well of the gas giant Pil orbits, instead of simply mining the carbonaceous asteroids present in the system."
```